src/presign/round_one.rs

```diff
@@ -26,9 +26,13 @@ pub(crate) struct Private {
 }
 #[derive(Debug, Serialize, Deserialize)]
 pub(crate) struct Public {
+    pub(crate) proof: PiEncProof,
+}
+
+#[derive(Debug, Serialize, Deserialize)]
+pub(crate) struct PublicBroadcast {
```

Comment:
If the struct declaration is being marked as `pub(crate)` then the fields can just be `pub`.